internal/utils/config/controlplane.go

```diff
@@ -0,0 +1,102 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/samber/lo"
+	corev1 "k8s.io/api/core/v1"
+
+	konnectv1alpha1 "github.com/kong/kubernetes-configuration/api/konnect/v1alpha1"
+)
+
+type kicInKonnectParams struct {
+	konnectAddress      string
+	controlPlaneID      string
+	tlsClientSecretName string
+}
+
+var kicInKonnectDefaults = func(params kicInKonnectParams) []corev1.EnvVar {
```

Comment:
Any reason to make this a `var` and not a `func`?

```suggestion
func kicInKonnectDefaults(params kicInKonnectParams) []corev1.EnvVar {
```

Reply:
leftover, moved to a function 👍🏻